app/controllers/organizations_controller.rb

```diff
@@ -9,13 +9,12 @@ class OrganizationsController < Orgs::Controller
   before_action :paginate_users_github_organizations, only: %i[new create]
   before_action :verify_user_belongs_to_organization, only: [:remove_user]
   before_action :set_filter_options,                  only: %i[search index]
+  before_action :set_filtered_organizations,          only: %i[search index]
 
   skip_before_action :ensure_current_organization,                         only: %i[index new create search]
   skip_before_action :ensure_current_organization_visible_to_current_user, only: %i[index new create search]
 
-  def index
-    @organizations = current_user.organizations.order(created_at: :desc).page(params[:page]).per(12)
-  end
+  def index; end
```

Comment:
Previously on a page load we were always loading all of the organizations regardless of whether or not params were provided. This change (setting `@organizations` using `set_filter_options` allows people to navigate directly to this page with a filtered URL and see only the classrooms that match those filters)